recipe/build.sh

```diff
@@ -8,11 +8,11 @@ mkdir build
 cd build
 
 ../configure \
-  CFLAGS="-I$PREFIX/include -I$PREFIX/include/asl" \
-  CXXFLAGS=" -m64 -I$PREFIX/include -I$PREFIX/include/asl" \
+  CFLAGS="-I$PREFIX/include -I$PREFIX/include/asl -I$PREFIX/include/mumps_seq" \
+  CXXFLAGS=" -m64 -I$PREFIX/include -I$PREFIX/include/asl -I$PREFIX/include/mumps_seq" \
   --with-blas-lib="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -lopenblas" \
   --with-asl-lib="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -lasl" \
-  --with-mumps-lib="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -ldmumps -lmumps_common -lpord -lmpiseq -lesmumps -lscotch -lscotcherr -lmetis -lgfortran" \
+  --with-mumps-lib="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -ldmumps_seq -lmumps_common -lpord -lmpiseq -lesmumps -lscotch -lscotcherr -lmetis -lgfortran" \
```

Comment:
The `_seq` suffix is on *all* the mumps libs. See [kwant for example](https://github.com/conda-forge/kwant-feedstock/pull/58/files#diff-56a4c8904b2b9cd576c57f93499b0b1fR4):

```suggestion
  --with-mumps-lib="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -ldmumps_seq -lmumps_common_seq -lpord_seq -lmpiseq_seq -lesmumps -lscotch -lscotcherr -lmetis -lgfortran" \
```